src/EigenvalueSolverTypes.f90

```diff
@@ -62,11 +62,19 @@ MODULE EigenvalueSolverTypes
 #else
 #include <finclude/petsc.h>
 #endif
-!petscisdef.h defines the keyword IS, and it needs to be reset
+
 #ifdef FUTILITY_HAVE_SLEPC
+!SLEPC version == PETSC version
+#if ((PETSC_VERSION_MAJOR>=3) && (PETSC_VERSION_MINOR>=6))
```

Comment:
Ha, its the last thing listed :)

And because the SLEPC version matches the PETSc version, I believe they made the same change.

> #### Fortran
>    * Fortran include files are now in include/petsc/finclude instead of include/finclude. Thus replace uses of #include "finclude/xxx.h" with #include "petsc/finclude/xxx.h"